velox/exec/RowContainer.cpp

```diff
@@ -282,6 +282,7 @@ char* RowContainer::initializeRow(char* row, bool reuse) {
     auto rows = folly::Range<char**>(&row, 1);
     freeVariableWidthFields(rows);
     freeAggregates(rows);
+    freeNextRowVectors(rows, false);
```

Comment:
Do we ever have the next row vectors in this case?

Reply:
This function is not called by HashBuild and HashProbe either. I added it here for consistency with other free functions. `freeNextRowVectors` function checks nextOffset , if it's 0, it will return directly.

Reply:
Can we just replace this with a check that it is always 0?

Reply:
Updated, thanks.